lib/go-tc/tovalidate/rules.go

```diff
@@ -15,9 +15,12 @@ package tovalidate
 import (
 	"errors"
 	"fmt"
+	validation "github.com/go-ozzo/ozzo-validation"
```

Comment:
This should come after the standard library imports

Reply:
Reordered imports in bbd66b5111.